apps/api-extractor/src/api/Extractor.ts

```diff
@@ -433,7 +433,7 @@ export class Extractor {
     extractorConfig: ExtractorConfig,
     messageRouter: MessageRouter
   ): void {
-    messageRouter.logInfo(
+    messageRouter.logVerbose(
       ConsoleMessageId.Preamble,
       `Analysis will use the bundled TypeScript version ${ts.version}`
```

Comment:
API Extractor installs its own version of the `typescript` package, and uses this instead of the TypeScript version that is used to compile your project. (There are several reasons for this, but the main reason is that the internal methods and AST tree change frequently between compiler releases, so we can't guarantee correct behavior with an untested version.)

Over the years, this caused a number of support issues, mostly:

- A particular syntax is parsed differently by a newer/older compiler, causing API Extractor to report an error for code that "compiles just fine"
- The compiler runtime declarations (e.g. `lib.esnext.d.ts`) have differences such that a newer/older compiler encounters errors

Either way, when these issues occur the investigation is much easier if (1) the developer is aware that API Extractor is using a different compiler version, and (2) whoever is diagnosing the issue can easily see what the two versions are.

> in parallel generating summarised types for a group of packages, the massage becomes annoying and there is not really a way to turn it off.

Hmm.... The `extractorMessageReporting` option in **api-extractor.json** is the normal way to suppress these messages:

https://github.com/microsoft/rushstack/blob/96e4e163cba3d52489e3488df7487e8abd2e425f/apps/api-extractor/src/api/ConsoleMessageId.ts#L14-L18

So I think the intended way to suppress it is like this:

**api-extractor.json**
```js
    "extractorMessageReporting": {
      "console-preamble": {
        "logLevel": "none"
      },
    },
```

However, this design seems slightly confusing to me:
- The English word "preamble" means "whatever stuff gets printed at the beginning," which as a user would make me wonder what else I might be accidentally suppressing.  (Looking at the code, it really is just that one message, but the name might better be like `ConsoleMessageId.BundledCompilerVersion`)
- `"logLevel": "none"` is not really what you asked for; you wanted to still see the message with `--verbose`.  There is not a `"logLevel": "verbose"` today, but maybe it would make sense to add one?

Reply:
@octogonz thank you for your explanation. I think what you said about having the ability the inform the TS version in use makes sense but it should be at least configurable somehow so we can deactivate it if we want. Before opening that PR I was looking for a way to do it with configuration and I dont think there is one. What you have suggested is not possible as the only possible keys to configure under `extractorMessageReporting` are listed at https://github.com/microsoft/rushstack/blob/master/apps/api-extractor/src/schemas/api-extractor-defaults.json#L51

That way it seems to me de only way to do something like this is by calling `logVerbose` (and adding a verbose level) and then control the default mode with 

```
"extractorMessageReporting": {
      "default": {
        "logLevel": "verbose" | "none"
      },
}
```

What do you think @octogonz ?

Reply:
@octogonz have you had the time to read through what I wrote?